### PR TITLE
Optimusrahul/registration update

### PR DESCRIPTION
**Pull request by OptimusRahul**
Hi,

I have completed this job challenge with 5 Projects, 3 Blogs. Kindly review, give me feedback and merge this pull request.
Thank you for giving such opportunity.

### CODE REVIEW

File: content/participants/ttjc@OptimusRahul/index.md [hunk 1]
```diff
@@ -47,8 +47,49 @@ It would be good if someone helps and suggest me the best practices while workin
...
+This is react based music player
+
+- You can find it [here](https://songify-player.netlify.app/)
+- Github repo can be found [here](https://github.com/OptimusRahul/Songify)
+
+### Open Source Project Finder
+
+This is javascript based website, that helps you to find all the open source project.
+
+- Github repo and preview can be found [here](https://github.com/OptimusRahul/Open-Source-Project-Finder)
```
Comment (gkanishk) [inline content/participants/ttjc@OptimusRahul/index.md:68]: Where's the hosted link? It's a website, right?

File: content/participants/ttjc@OptimusRahul/index.md [hunk 2]
```diff
@@ -47,8 +47,49 @@ It would be good if someone helps and suggest me the best practices while workin
...
+
+### GitDesk
+
+This is a desktop based application, which gives you a new and fast way to use github
+
+- Github repo and preview can be found [here](https://github.com/OptimusRahul/GitDesk)
```
Comment (gkanishk) [inline content/participants/ttjc@OptimusRahul/index.md:74]: @tanaypratap here's an electron app, should have a look!!

File: content/participants/ttjc@OptimusRahul/index.md (hunk 1 above)
Comment (OptimusRahul) [reply content/participants/ttjc@OptimusRahul/index.md:68]: Yes, it's website, I was facing issue to deploy it over netlify, so I had attached the preview of the project. Now I have finally hosted it over heroku.
link - https://open-source-project-finder.herokuapp.com/
-[App](https://open-source-project-finder.herokuapp.com/)

Should I raise a pull request again?

**Comment by gkanishk** [comment]
Awesome!! You raise a PR for the finalist now refer [faq](https://2020.teamtanay.jobchallenge.dev/finalist-entry/) or #935

**Pull request merged**